app/views/results/_early_summary.html.slim

```diff
@@ -0,0 +1,45 @@
+- data = @model.summary_data(section)
```

Comment:
i added a new early summary section for now as looking at the other was making my head spin...but this way of doing it isn't very sustainable for a check at disposable too - thinking we can make a ticket to refactor our summary section when we start doing disposable early checks too maybe?

Reply:
sounds like a good midway point. we dont know when disposable work will be done, better to get this feature delivered than trying to future proof it too early